Automattic/course
# Add footer template and pattern

Implements https://github.com/Automattic/sensei-theme/issues/27

## Changes proposed

Added pattern and template for footer with dynamic site title font size and URLs

**Note:** The dynamic font size is implemented only for the frontend.
## Testing instructions

- Go to any page in the frontend
- Make sure the footer looks like the design
- Try with different site titles
- Make sure the site title showing part in the bottom holds up looks alright in different screen sizes

## Screenshot

**Desktop**
<img width="1122" alt="Screen Shot 2022-11-03 at 10 39 05 PM" src="https://user-images.githubusercontent.com/6820724/199782350-ef412b8b-8568-4867-8350-f50b6f036516.png">

**Desktop with long title**
<img width="1075" alt="Screen Shot 2022-11-03 at 10 41 08 PM" src="https://user-images.githubusercontent.com/6820724/199782440-3d84db82-1ac7-4444-b0dc-1c3f4a0b9d94.png">

**Mobile**
<img width="461" alt="Screen Shot 2022-11-03 at 10 39 57 PM" src="https://user-images.githubusercontent.com/6820724/199782517-1eb9a151-e987-4857-9915-2866f2668b56.png">
